pyproject.toml

```diff
@@ -1,3 +1,13 @@
 [build-system]
 requires = ["setuptools"]
-build-backend = "setuptools.build_meta"
\ No newline at end of file
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "s2-python"
+description = "S2 Protocol Python Implementation"
+version = "0.5.0"
+
+[project.optional-dependencies]
+ws = ["websockets"]
+fastapi = ["fastapi"]
+flask = ["Flask"]
```

Comment:
Some of this was already defined in `setup.cfg`. Let's keep it there (see [this related PR](https://github.com/flexiblepower/s2-python/pull/103)).